### DISCUSSION TITLE
Is it production ready?

### DISCUSSION THREAD

**Discussion by dirkarnez** [ent/ent#2805] [Q&A] [answered]
As title, thank you!

**Comment by frederikhors** [comment]
Are you joking?

**Comment by giautm** [comment, marked as answer]
hello @dirkarnez, welcome to our community. Please join [the discord server](https://discord.gg/qZmPgTE6RX) if you have further questions.

https://www.linuxfoundation.org/blog/ent-joins-the-linux-foundation/
